.github/workflows/publish-to-pypi.yml

```diff
@@ -0,0 +1,40 @@
+name: Publish 📦 to PyPi 🚀
+
+on:
+  push:
+    tags:
+      - "[0-9]+.[0-9]+.[0-9]+*"
```

Comment:
Usually, our version start with are prefixed with a 'v', like 'v0.9.2'. Can you include the 'v' here?

Reply:
ah, no, sorry I was wrong about the versioning, the `.` is not a wildcard character here